src/main/java/ro/msg/learning/shop/controllers/ProductController.java

```diff
@@ -21,8 +22,8 @@ public ProductController(ProductService productService, ProductMapper productMap
     }
 
     @PostMapping(value = "")
-    public void create(@RequestBody ProductDTO productDTO) {
-        productService.create(productMapper.toProduct(productDTO));
+    public Product create(@RequestBody ProductDTO productDTO) {
```

Comment:
Nice for returning something, but the frontend eats DTOs, not entities. Return a DTO please. Entities are representations of our db tables; The front end sends a dto and should receive a dto
💡 for further reading on the subject:
https://www.linkedin.com/pulse/difference-between-entity-dto-what-use-instead-omar-ismail/?trk=pulse-article_more-articles_related-content-card